lib/auth/auth.go

```diff
@@ -57,12 +57,13 @@ import (
 
 	"github.com/coreos/go-oidc/oauth2"
 	"github.com/coreos/go-oidc/oidc"
-	"github.com/gravitational/trace"
 	"github.com/jonboulle/clockwork"
 	"github.com/pborman/uuid"
 	"github.com/prometheus/client_golang/prometheus"
 	saml2 "github.com/russellhaering/gosaml2"
 	"golang.org/x/crypto/ssh"
+
+	"github.com/gravitational/trace"
```

Comment:
nit: Why'd you separate this from the import group? I thought `trace` was supposed to be separated from the main teleport import group.

Reply:
I mis-configured `gopls` import grouping in my editor, reverted :)